server/tournaments/index.js

```diff
@@ -83,7 +84,7 @@ class Tournament extends Rooms.RoomGame {
 		this.playerCap = (playerCap ? parseInt(playerCap) : Config.tourdefaultplayercap) || 0;
 
 		/** @type {string} */
-		this.format = formatId;
+		this.format = name || formatId;
```

Comment:
This seems like a bad name for this property.

Reply:
What name would you prefer?

Reply:
Also, the property is called `format` in the update message for whatever reason, and I'd _really_ prefer to not change that.

Reply:
It's fine not to change the property in the protocol, but at least changing it here seems valid. Something like `this.name`?

Reply:
But then you need to send updates like ``{format: this.name}``, which looks weird and is more confusing IMO.

Reply:
To me, it's much more obvious that `{format: this.name}` isn't sending a format, than that `{format: this.format}` isn't sending a format.

Reply:
You should also comment it, to make it extra clear.

    // note: this is not a format, it's just called format for historical reasons
    {format: this.name}